build/chip/chip_test_suite.gni

```diff
@@ -78,6 +96,115 @@ template("chip_test_suite") {
     invoker.sources += invoker.test_sources
   }
 
+  if (enable_efr32_build_per_suite || enable_efr32_build_per_testsource) {
+    # Since target_name is usually "tests", we need a unique name to use in the executable target.
+    if (defined(invoker.output_name)) {
+      # Get it from the suite's output_name.
+      _unique_suite_name = invoker.output_name
+    } else {
+      # Some test suites don't have output_name defined. Use the first source file.
+      _unique_suite_name = string_replace(string_join("-", invoker.test_sources), ".cpp", "")
+    }
+  }
+
+  print("++++ SUITE ${_unique_suite_name}")
+  if (enable_efr32_build_per_suite) {
```

Comment:
This `if` branch is not executed unless we change the debug flags to enable per-suite binaries instead of per-test binaries.